ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -103,10 +103,17 @@ - (NSString*) deviceName
                               @"iPhone9,4" :@"iPhone 7 Plus",   // (model A1784 | Global)
                               @"iPhone10,3":@"iPhone X",        // (model A1865, A1902)
                               @"iPhone10,6":@"iPhone X",        // (model A1901)
+                              @"iPhone11,8":@"iPhone XR",       // (model A1882, A1719)
```

Comment:
Is it alright for there to be duplicate entries?

Reply:
Ah, I should've said "duplicate keys", my bad. Line 106 has an entry for `@"iPhone11,8"` and line 116 has a duplicate entry for the same key.

Reply:
@mattlubner I see what you are saying. Likely somethign weird with the merge. I'll clean that up